Sources/MusicXML/Complex Types/NoteSize.swift

```diff
@@ -12,10 +12,11 @@
 /// large size. The text content represents the numeric percentage. A value of 100 would be
 /// identical to the size of a regular note as defined by the music font.
 public struct NoteSize {
-    public let value: NonNegativeDecimal
+    // FIXME: Use `NonNegativeDecimal` if that is wise.
+    public let value: Double
```

Comment:
I think the benefit of typed value is that we can add a validation layer in its initialization to fend against invalid inputs, also the `ExpressibleBy*` protocol enables us to use literals to reduce typing.

I think lots of typed numerals are already a crucial part of the codebase like `DegreeValue`, `DegreeAlter`,  `Tenths`, I lean on the side of strong typing the values.

Reply:
Definitely agree. I think this could be a single-sweep in a PR.